deploy/sdk/src/dynamo/sdk/request_tracing.py

```
@@ -0,0 +1,341 @@
+                            i
+                            for i, p in enumerate(sig.parameters.values())
+                            if p.name == param_name
+                        ),
+                        None,
+                    )
+                    if param_idx is not None:
+                        adj_idx = param_idx - 1  # skip `self`
+                        if 0 <= adj_idx < len(args):
+                            request_id = args[adj_idx]
+
+                            # Replace value in the *args* tuple instead of duplicating it in **kwargs**
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)